src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -104,24 +102,6 @@ public Consent describeConsentForDAR(@Auth AuthUser authUser, @PathParam("id") S
         return c;
     }
 
-
-    @GET
-    @Produces("application/json")
-    @Path("/manage")
```

Comment:
This endpoint is still in use here: https://github.com/DataBiosphere/consent/blob/develop/automation/src/test/scala/org/broadinstitute/dsp/consent/requests/Requests.scala#L194

Let's chat about how to update the automation tests to point to the right url